commander/main.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import argparse
+
+import settings
+from engine import Engine
+
+
+def main():
+    connection_string = settings.DEFAULT_SERIAL_PORT
+    baud_rate = settings.DEFAULT_BAUD_RATE
+
+    if arg_options.connect:
+        connection_string = arg_options.connect
+    if arg_options.baud_rate:
+        baud_rate = arg_options.baud_rate
+
+    engine = Engine(connection_string, baudrate=baud_rate)
```

Comment:
Shouldn't the `connection_string` and `baud_rate` be params for the `connect` instead of belong to the `Engine`?

Reply:
Would be an option but I think that in this case it makes sense to be in the initialization of the `Engine` class because those attributes are essential for it to work.